pkg/systemd/helpers.js

```diff
@@ -0,0 +1,75 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2019 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+import $ from "jquery";
+import cockpit from "cockpit";
+
+export function set_page_link(element_sel, page, text) {
```

Comment:
This function is only being used in graphs.js. It's not something that we want to propagate to other pages IMHO, so maybe it could just move into graphs.js, and we'll port it to proper React/non-jquery later?

Reply:
Moved to graphs.js